app/build.gradle

```diff
@@ -56,14 +56,15 @@ dependencies {
       'androidx.test.espresso:espresso-core:3.2.0',
       'androidx.test.ext:junit:1.1.1',
       'com.google.truth:truth:0.43',
-      'org.robolectric:robolectric:4.3',
+      'org.robolectric:robolectric:4.3'
```

Comment:
Super minor nit here & below: prefer keeping the trailing comma just to make it easier to add other dependencies later (or move them around).

Reply:
added comma